cypress/integration/registration.spec.js

```diff
@@ -0,0 +1,118 @@
+import { generateStringWithRandomPrefix } from '../support/utils';
+import Navbar from '../page-objects/components/Navbar';
+import Registration from '../page-objects/pages/Registration';
+
+
+describe('User Registration Process', () => {
+
+    describe('User Registration Valid Scenario', () => {
+        before(function () {
+        cy.visit('http://localhost/kino/')
+        Navbar.clickOnLogin()
+        cy.fixture('registration.json').as('users');
+       })
+
+     it('Should regiser user properly', function () {
+        const name = generateStringWithRandomPrefix(
+            this.users['valid_name'].allowed
+        );
+        const lastname = generateStringWithRandomPrefix(
+            this.users['valid_lastname'].allowed
+        );
+        const password = this.users['valid_password'].allowed;
+        const email = generateStringWithRandomPrefix(
+            this.users['valid_email'].allowed
+        );
+
+        cy.fillRegisterMandatoryFields(name, lastname, email, password);
+
+        cy.contains('Registracija').click();
+
+        cy.get('.alert-success').should('contain', Registration.selectors.SUCCESS_MSG);

```

Comment:
In this case the Registration.selectors is used. How about using it in other cy.get() as selectors. The success messages of each process can be also specific list of objects in specific page. 

The usage of POM (Page Object Model) is actually not recommended to use in Cypress. Check this out: https://www.cypress.io/blog/2019/01/03/stop-using-page-objects-and-start-using-app-actions/